components/application-mgt/org.wso2.carbon.identity.application.common/pom.xml

```diff
@@ -18,7 +18,7 @@
     <parent>
         <groupId>org.wso2.carbon.identity.framework</groupId>
         <artifactId>application-mgt</artifactId>
-        <version>5.18.31</version>
+        <version>5.18.32-SNAPSHOT</version>
```

Comment:
Any reason for version changes in the pom files?

Reply:
Rebased and fixed.